Source/WTF/wtf/Vector.h

```diff
@@ -1932,7 +1932,7 @@ template<typename DestinationItemType, typename Collection>
 inline auto moveToVectorOf(Collection&& collection) -> Vector<DestinationItemType>
 {
     return WTF::map(collection, [] (auto&& v) -> DestinationItemType {
-        return WTFMove(v);
+        return std::forward<DestinationItemType>(v);
```

Comment:
The use of `v` for this local variable name, which is not a vector, is unnecessarily confusing. I suggest renaming to `item`.